test/midcom/helper/imagepopup/handler/uploadTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * @package openpsa.test
+ * @author CONTENT CONTROL http://www.contentcontrol-berlin.de/
+ * @copyright CONTENT CONTROL http://www.contentcontrol-berlin.de/
+ * @license http://www.gnu.org/licenses/gpl.html GNU General Public License
+ */
+
+/**
+ * OpenPSA testcase
+ *
+ * @package openpsa.test
+ */
+class midcom_helper_imagepopup_handler_uploadTest extends openpsa_testcase
+{
+    protected static $_images;
+    protected static $_tmp_folder;
+    protected static $_tmp_names;
+    
+    public static function setupBeforeClass()
+    {
+        self::$_images = array();
+        self::$_tmp_names = array();
+        // We need two images to run tests
+        self::$_images = self::create_images(2);
+    }
+    
+    public function testHandler_upload()
+    {
+        midcom::get()->auth->request_sudo('midcom.helper.imagepopup');
+        $node = self::get_component_node('net.nehmer.static');
+        $_FILES = self::$_images;
+        
+        // Test images really exists ?
+        foreach(self::$_tmp_names as $temp_name)
```

Comment:
isn't `self::$_tmp_names` empty at this point? i.e. the loop never runs, or did I miss something?